bears/documentation/DocumentationStyleBear.py

```diff
@@ -0,0 +1,118 @@
+from coalib.bearlib.languages.documentation.DocumentationComment import (
+    DocumentationComment)
+from coalib.bearlib.languages.documentation.DocstyleDefinition import (
+    DocstyleDefinition)
+from coalib.bearlib.languages.documentation.DocumentationExtraction import (
+    extract_documentation)
+from coalib.bears.LocalBear import LocalBear
+from coalib.results.Diff import Diff
+from coalib.results.Result import Result
+from coalib.results.TextRange import TextRange
+
+
+class DocumentationStyleBear(LocalBear):
+    LANGUAGES = {language for docstyle, language in
+                 DocstyleDefinition.get_available_definitions()}
+    AUTHORS = {'The coala developers'}
+    AUTHORS_EMAILS = {'coala-devel@googlegroups.com'}
+    LICENSE = 'AGPL-3.0'
+    ASCIINEMA_URL = 'https://asciinema.org/a/7sfk3i9oxs1ixg2ncsu3pym0u'
```

Comment:
@hemangsk hmm its correct I guess.
![screenshot from 2017-06-04 21-43-18](https://cloud.githubusercontent.com/assets/13048113/26763330/c53f9200-4940-11e7-9eb0-a83da487d159.png)
Although, I appreciate you checked that :smile: